src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppEcsFargate/AppStack.cs

```diff
@@ -19,121 +17,42 @@ public class AppStack : Stack
-                Cpu = settings.TaskCpu,
-                MemoryLimitMiB = settings.TaskMemory
-            });
+            // Create the recipe defined CDK construct with all of its sub constructs.
+            var generatedRecipeConstruct = new RecipeConstruct(this, recipeConfiguration, props);
 
-            if (string.IsNullOrEmpty(recipeConfiguration.ECRRepositoryName))
-                throw new InvalidOrMissingConfigurationException("The provided ECR Repository Name is null or empty.");
-
-            var ecrRepository = Repository.FromRepositoryName(this, "ECRRepository", recipeConfiguration.ECRRepositoryName);
-            var container = taskDefinition.AddContainer("Container", new ContainerDefinitionOptions
-            {
-                Image = ContainerImage.FromEcrRepository(ecrRepository, recipeConfiguration.ECRImageTag)
-            });
-
-            container.AddPortMappings(new PortMapping
-            {
-                ContainerPort = 80,
-                Protocol = Protocol.TCP
-            });
-
-            var ecsLoadBalancerAccessSecurityGroup = new SecurityGroup(this, "WebAccessSecurityGroup", new SecurityGroupProps
-            {
-                Vpc = vpc,
-                SecurityGroupName = $"{recipeConfiguration.StackName}-ECSService"
-            });
-
-            var ecsServiceSecurityGroups = new List<ISecurityGroup>();
-            ecsServiceSecurityGroups.Add(ecsLoadBalancerAccessSecurityGroup);
-
-            if (!string.IsNullOrEmpty(settings.AdditionalECSServiceSecurityGroups))
-            {
-                var count = 1;
-                foreach (var securityGroupId in settings.AdditionalECSServiceSecurityGroups.Split(','))
-                {
-                    ecsServiceSecurityGroups.Add(SecurityGroup.FromSecurityGroupId(this, $"AdditionalGroup-{count++}", securityGroupId.Trim(), new SecurityGroupImportOptions
-                    {
-                        Mutable = false
-                    }));
-                }
-            }
+            // Create additional CDK constructs here. The recipe's constructs can be accessed as properties on
+            // the generatedRecipeConstruct variable.
+        }
 
-            new ApplicationLoadBalancedFargateService(this, "FargateService", new ApplicationLoadBalancedFargateServiceProps
-            {
-                Cluster = cluster,
-                TaskDefinition = taskDefinition,
-                DesiredCount = settings.DesiredCount,
-                ServiceName = settings.ECSServiceName,
-                AssignPublicIp = settings.Vpc.IsDefault,
-                SecurityGroups = ecsServiceSecurityGroups.ToArray()
-            });
+        /// <summary>
+        /// This method can be used to customize the properties for CDK constructs before creating the constructs.
+        ///
+        /// The pattern used in this method is to check to evnt.ResourceLogicalName to see if the CDK construct about to be created is one

```

Comment:
typo: "check the event.ResourceLogicalName" instead of "check to evnt.ResourceLogicalName"

Reply:
On purpose because `event` doesn't work as a variable name since it is a reserved word in .NET.